packages/design-tokens/README.md

```diff
@@ -2,142 +2,94 @@
 
 > Design tokens for Spirit Design System.
 
-⚠️ Spirit design tokens are managed with and generated by [Supernova]. DO NOT EDIT MANUALLY!
+⚠️ Spirit design tokens are managed with and generated by [Supernova][supernova]. DO NOT EDIT MANUALLY!
 
 ## Table of Contents
 
 1. [Available Design Tokens](#available-design-tokens)
+   1. [Global Tokens](#global-tokens)
+   2. [Theme Tokens](#theme-tokens)
 2. [Install](#install)
 3. [Basic Usage](#basic-usage)
-4. [`@tokens` API](#tokens-api)
+   1. [In JavaScript](#in-javascript)
+4. [Rebranding Spirit](#rebranding-spirit)
 5. [FAQ](#faq)
+6. [License](#license)
 
 ## Available Design Tokens
 
-| Category      | Supernova | Figma | Sass                 |
-| ------------- | --------- | ----- | -------------------- |
-| 🖼 Borders    | ✅        | ❌    | [`_borders.sass`]    |
-| 🎨 Colors     | ✅        | ✅    | [`_colors.sass`]     |
-| 🖌️ Gradients  | ✅        | ✅    | [`_gradients.sass`]  |
-| 📏️ Measures  | ✅        | ❌    | [`_measures.sass`]   |
-| ⚙️ Other      | ✅        | ❌    | [`_other.sass`]      |
-| 🎱 Radii      | ✅        | ❌    | [`_radii.sass`]      |
-| ⛱ Shadows    | ✅        | ✅    | [`_shadows.sass`]    |
-| 🔠 Typography | ✅        | ✅    | [`_typography.sass`] |
+Tokens are split into two categories: Global and Themes. Global tokens are
+shared across all themes, while theme tokens are specific to a particular theme.
 
-## Install
+They are managed in Figma and exported to Supernova.
 
-🙋🏻‍♂️ **Hold on! Do you already use [`spirit-web`]?** Then you don't need to
-install this package because `spirit-design-tokens` is installed automatically
-as a dependency of [`spirit-web`].
+All content in `src` is generated by Supernova and should not be edited manually.
 
-If you want to use just `spirit-design-tokens` alone in your project, run:
+In `scss` directory are files `@global.scss` and `@themes.scss` which contain links to all available tokens.
 
-```shell
-yarn add @lmc-eu/spirit-design-tokens
-```
+### Global Tokens
 
-or
+The category consist of these groups:
 
-```shell
-npm install --save @lmc-eu/spirit-design-tokens
-```
+- Borders
```

Comment:
I want those nice emojis from the table back again 🙏 :-)

Reply:
😑